include/openspace/navigation/orbitalnavigator.h

```diff
@@ -186,8 +197,18 @@ class OrbitalNavigator : public properties::PropertyOwner {
 
     properties::BoolProperty _followAnchorNodeRotation;
     properties::FloatProperty _followAnchorNodeRotationDistance;
+    properties::BoolProperty _enableMinimumAllowedDistanceLimit;
     properties::FloatProperty _minimumAllowedDistance;
 
+    struct LimitZoomOut : public properties::PropertyOwner {
+        LimitZoomOut();
+
+        properties::BoolProperty enabled;
+        properties::FloatProperty maximumAllowedDistance;
```

Comment:
Currently, the maximum allowed distance is given in meters and the minimum allowed distance is given in x * interaction sphere.
It would be nice to harmonize that to use either for both of them.  Personally I think I would tend to using the interaction sphere rather than meters?